postView.html

```diff
@@ -1,12 +1,324 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/styles/index.css" />
+    <script src="https://use.fontawesome.com/350296bd22.js"></script>
+    <title>Home-Screen</title>
+  </head>
+
+  <body>
+    <header class="header--post-View px-1-5rem">
+      <div class="icon-container">
+        <svg
+          width="14"
+          height="12"
+          viewBox="0 0 14 12"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M5.66667 10.6666L1 5.99992M1 5.99992L5.66667 1.33325M1 5.99992H13"
+            stroke="white"
+            stroke-width="2"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+          />
+        </svg>
+      </div>
+      <h1 class="text--uppercase">post</h1>
+    </header>
+
+    <main class="main">
+      <section class="post post--no-topline">
+        <div class="avatar">
+          <img
+            class="avatar__image"
+            src="https://source.unsplash.com/100x100/?photographer"
+            alt=""
+          />
+          <h2 class="avatar__heading">Tony Monta</h2>
+          <span class="avatar__timespan">1h ago</span>
+        </div>
+        <span class="post__burger-menu"></span>
+        <div class="post__text">
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Quae
+            possimus aut sed accusamus est obcaecati non vel dolore,
+            reprehenderit dolor
+          </p>
+        </div>
+        <div class="post__icons">
+          <span class="post__icon--thumb mr-1rem">2245</span>
+          <span class="post__icon--bubble mr-1rem">45</span>
+          <span class="post__icon--share mr-1rem">124</span>
```

Comment:
This numbers will be read out - and the user won't know why. You need text to provide accessible content.